WeakAurasOptions/AuthorOptions.lua

```diff
@@ -975,6 +975,35 @@ typeControlAdders = {
       step = 1
     }
   end,
+  media = function(options, args, data, order, prefix, i)
+    local option = options[i]
+    args[prefix .. "mediaType"] = {
+      type = "select",
+      width = WeakAuras.normalWidth,
+      name = name(option, "mediaType", L["Media Type"]),
+      desc = desc(option, "mediaType"),
+      values = OptionsPrivate.Private.shared_media_types,
+      order = order(),
+      get = get(option, "mediaType"),
+      set = set(data, option, "mediaType")
+    }
+    args[prefix .. "default"] = {
+      type = "select",
+      width = WeakAuras.normalWidth,
+      name = name(option, "default", L["Default"]),
+      desc = desc(option, "default"),
+      values = function()
+        if option.mediaType == "sound" then
+          return OptionsPrivate.Private.sound_types
+        else
+          return AceGUIWidgetLSMlists[option.mediaType]
+        end
+      end,
```

Comment:
This sort sounds like in other widgets
```suggestion
      end,
      sorting = function()
        if option.mediaType == "sound" then
          return OptionsPrivate.Private.SortOrderForValues(OptionsPrivate.Private.sound_types)
        else
          return nil
        end
      end,
```